frontend/src/metabase/components/UserAvatar/UserAvatar.jsx

```diff
@@ -29,7 +29,20 @@ function initial(name) {
 }
 
 function userInitials(user) {
-  return user ? initial(user.first_name) + initial(user.last_name) : null;
+  // https://user-images.githubusercontent.com/1937582/171615363-40bb26ca-6d81-4d0f-b144-2d679d2ede77.png
+  if (user) {
+    return nameInitials(user) || emailInitials(user);
```

Comment:
Following [the discussion on Slack](https://metaboat.slack.com/archives/C03E8MQJZBM/p1654766630306639?thread_ts=1654759255.159039&cid=C03E8MQJZBM), uses the first 2 letters from an email as an initial name.

<img src="https://user-images.githubusercontent.com/1937582/171615363-40bb26ca-6d81-4d0f-b144-2d679d2ede77.png" />